src/sentry/static/sentry/less/forms.less

```diff
@@ -154,7 +154,14 @@ label {
 }
 
 .has-error .form-control {
-  border-color: @red-dark;
+  border-color: #c9c0d1;
+  box-shadow: inset 0 2px 0 rgba(0, 0, 0, 0.04);
+}
```

Comment:
Can this negatively impact forms in other parts of the application?

Reply:
I believe not, because:

`has-error `is currently used in:

`booleanField.jsx` - that is being used by `GenericField.jsx` that is being only in getSentry (Anyway I believe it's gonna be fine as well)
`formField` - this is the one that I updated and it's fine 😉
`MultipleCheckboxField` - this is being used only in `OrganizationApiKeyDetails.jsx ` and it's fine
`InviteMember.jsx` is fine: 
![image](https://user-images.githubusercontent.com/29228205/80716550-0f36e600-8af8-11ea-835d-51a88c516ae7.png)

Reply:
Just a heads up, `GenericField` is generally used by our plugins where the fields are declared on the backend so grep will not show its true usage.

My other concern here would be if the has error color can be mismatched by other things that use `@red-dark` as the "error" color.